### PR TITLE
NER : Add output like html tag

### PR DESCRIPTION
**Pull request by wannaphongcom**
Add output like html tag (ThaiNameTagger).

```python
>>> from pythainlp.tag.named_entity import ThaiNameTagger
>>> ner = ThaiNameTagger()
>>> ner.get_ner("วันที่ 15 ก.ย. 61 ทดสอบระบบเวลา 14:49 น.",tag=True)
'วันที่ <DATE>15 ก.ย. 61</DATE> ทดสอบระบบเวลา <TIME>14:49 น.</TIME>'
```

### CODE REVIEW

File: pythainlp/tag/named_entity.py [hunk 1]
```diff
@@ -145,13 +149,29 @@ def get_ner(
...
+            return self.sent
+        elif pos:
             return [
                 (self.__pos_tags[i][0], self.__pos_tags[i][1], data)
                 for i, data in enumerate(self.__y)
             ]
+        else:
+            return self.sent_ner
 
-        return [(self.__pos_tags[i][0], data) for i, data in enumerate(self.__y)]
```
Comment (heytitle) [inline pythainlp/tag/named_entity.py]: It would be good if we have tests for this change.

File: tests/test_tag.py [hunk 2]
```diff
@@ -87,6 +87,105 @@ def test_ner(self):
...
+        self.assertEqual(
+            ner.get_ner(
+                "เบอร์โทรศัพท์ 00-120-1100",
+                tag=True
+            ),
+            "เบอร์โทรศัพท์ <PHONE>00-120-1100</PHONE>")
+
+        self.assertEqual(
+            ner.get_ner(
+                "อาจารย์เอกพล ประจำคณะวิสกรรมศาสตร์ ",
```
Comment (heytitle) [inline tests/test_tag.py]: typo?

**Comment by heytitle** [comment]
Apart from the typo, the PR looks good to me.

File: tests/test_tag.py (hunk 2 above)
Comment (artificiala) [reply tests/test_tag.py]: Oh, yes. Thanks for checking the typos.

**Comment by wannaphongcom** [comment]
if this pull request is ready, you can merge it.

**Pull request merged**